calico_kubernetes/calico_kubernetes.py

```diff
@@ -89,6 +89,24 @@ def delete(self, namespace, pod_name, docker_id):
             logger.warning("Cannot remove profile %s; Profile cannot be found.",
                            self.profile_name)
 
+    def status(self, namespace, pod_name, docker_id):
+        self.namespace = namespace
+        self.pod_name = pod_name
+        self.docker_id = docker_id
+
+        # Obtain IP Address of container
+        ip = self._read_docker_ip()
+
+        json_dict = {
+            "apiVersion": "v1beta1",
+            "kind": "PodNetworkStatus",
+            "ip": str(ip)
+        }
+
+        logger.info("Writing json dict to stdout: \n%s", json.dumps(json_dict))
```

Comment:
This log is more useful if it says its writing the pod status to stdout.

So something like "Writing status for pod %s: %s"